### DISCUSSION TITLE
Channel Emote Popup Box Autocomplete Priority

### DISCUSSION THREAD

**Discussion by davelife** [Chatterino/chatterino2#3084] [Ideas]
I've tried looking in issues and the discussions here and I didn't find an exact match, so sorry if I didn't find it. 

I think an option that would prioritize the autocompletion emote list with that particular channel's emotes would be useful.

I'm talking about the little popup box that shows up.

For example, typing :hi   I'd like the first option be that particular channel's emote with 'hi' in the name. 

Said another way, if I'm in channel A, I'd like channelaHi to show up first, and when I'm in channel B have channelbHi show up first.

Right now, sometimes I get emoji and other emotes first - depending what I type.

**Comment by apa420** [comment]
#894

**Reply by davelife** [reply]
I saw that thread, but in my confused brain, I felt having a channel's emotes prioritized would be different then selecting specific individual ones to be prioritized - they are already prioritized in the emote selector. I felt this was different suggestion, provided consistency, and might be within the scope of the project, sorry.

**Reply by apa420** [reply]
In my opinion your "issue" is way too niche to most users to the chat client and I don't see it worth implementing because of little usecount it would get.
Most people would use the feature suggested in the issue or prefix the emote with maybe the first letter of the streamer when autocompleting.

Eg.
aHi
bHi
